include/ofi_util.h

```diff
@@ -130,6 +130,8 @@
 	FI_INFO_STRING(provider, prov->name, user->name, "Supported",	\
 		       "Requested")
 
+#define ofi_div_ceil(a, b) ((a + b - 1) / b)
```

Comment:
Can you move this to ofi.h?  That seems to be the best place currently, where we have a few other math related functions.